Strings.ipynb

```diff
@@ -0,0 +1,815 @@
+       "['__add__',\n",
+       " '__class__',\n",
+       " '__contains__',\n",
+       " '__delattr__',\n",
+       " '__dir__',\n",
+       " '__doc__',\n",
+       " '__eq__',\n",
+       " '__format__',\n",
+       " '__ge__',\n",
+       " '__getattribute__',\n",
+       " '__getitem__',\n",
+       " '__getnewargs__',\n",
+       " '__gt__',\n",
+       " '__hash__',\n",
+       " '__init__',\n",
+       " '__init_subclass__',\n",
+       " '__iter__',\n",
+       " '__le__',\n",
+       " '__len__',\n",
+       " '__lt__',\n",
+       " '__mod__',\n",
+       " '__mul__',\n",
+       " '__ne__',\n",
+       " '__new__',\n",
+       " '__reduce__',\n",
+       " '__reduce_ex__',\n",
+       " '__repr__',\n",
+       " '__rmod__',\n",
+       " '__rmul__',\n",
+       " '__setattr__',\n",
+       " '__sizeof__',\n",
+       " '__str__',\n",
+       " '__subclasshook__',\n",
+       " 'capitalize',\n",
+       " 'casefold',\n",
+       " 'center',\n",
+       " 'count',\n",
+       " 'encode',\n",
+       " 'endswith',\n",
+       " 'expandtabs',\n",
+       " 'find',\n",
+       " 'format',\n",
+       " 'format_map',\n",
+       " 'index',\n",
+       " 'isalnum',\n",
+       " 'isalpha',\n",
+       " 'isascii',\n",
+       " 'isdecimal',\n",
+       " 'isdigit',\n",
+       " 'isidentifier',\n",
+       " 'islower',\n",
+       " 'isnumeric',\n",
+       " 'isprintable',\n",
+       " 'isspace',\n",
+       " 'istitle',\n",
+       " 'isupper',\n",
+       " 'join',\n",
+       " 'ljust',\n",
+       " 'lower',\n",
+       " 'lstrip',\n",
+       " 'maketrans',\n",
+       " 'partition',\n",
+       " 'removeprefix',\n",
+       " 'removesuffix',\n",
+       " 'replace',\n",
+       " 'rfind',\n",
+       " 'rindex',\n",
+       " 'rjust',\n",
+       " 'rpartition',\n",
+       " 'rsplit',\n",
+       " 'rstrip',\n",
+       " 'split',\n",
+       " 'splitlines',\n",
+       " 'startswith',\n",
+       " 'strip',\n",
+       " 'swapcase',\n",
+       " 'title',\n",
+       " 'translate',\n",
+       " 'upper',\n",
+       " 'zfill']"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Few Examples:\n",
+    "print(\"Example of few methods: \\n\")\n",
+    "line = \"example methods\"\n",
+    "print(line, \"-> upper() ->\", line.upper())\n",
+    "\n",
+    "# Notice escape character\n",
+    "print(line, \"-> replace(\\\"example\\\", \\\"useful\\\") ->\", line.replace(\"example\", \"useful\"))\n",
+    "\n",
+    "# Notice tab character\n",
+    "print(\"    \\tI have a tab and spaces\", \"-> lstrip() ->\", \"    \\tI don't\".lstrip())\n",
+    "\n",
+    "# Methods. Notice the new line character. \n",
+    "print(\"\\nAvailable Methods: \")\n",
+    "dir(line)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d8125b0f-0070-4eab-a464-257284fcbbbe",
+   "metadata": {},
+   "source": [
+    "____\n",
+    "## String Format ##\n",
```

Comment:
I think we're missing the _var1=1;f"var1: {var1}"_ kind of formatting here. Maybe an example converting an _int_ or even multiple types to string could be helpful. Also a couple prints.

Reply:
You mean value conversion using this kind of formatting?

Reply:
Yes, for instance, instead of concatenate with _+_, and using _f"{}"_ instead of _.format()_:
```python
var1 = 1
var2 = 3.14
string1 = f"The value of {var1} is not directly related to {var2}."
```